### PR TITLE
Fix checkbox and radio button icons

### PR DESCRIPTION
**Pull request by devkanro**
This pr fix presentation of checkbox and radio button by moving icon to path that same as IDEA and the PaletteMapper can work fine with standalone

### CODE REVIEW

File: core/src/main/kotlin/org/jetbrains/jewel/IconMapper.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import androidx.compose.ui.res.ResourceLoader
+
+interface IconMapper {
+
+    fun mapPath(originalPath: String, iconData: IntelliJThemeIconData, resourceLoader: ResourceLoader): String
+}
+
+object IntelliJIconMapper : IconMapper {
+
+    private const val VERBOSE = true
```
Comment (Walingar) [inline core/src/main/kotlin/org/jetbrains/jewel/IconMapper.kt:12]: Let's disable VERBOSE by default

File: ide-laf-bridge/build.gradle.kts [hunk 2]
```diff
@@ -7,7 +7,7 @@ plugins {
 }
 
 dependencies {
-    api(projects.intUi.intUiStandalone) {
+    api(projects.intUi.intUiCore) {
```
Comment (Walingar) [inline ide-laf-bridge/build.gradle.kts:10]: Good point!

**Review by Walingar — APPROVED** [review]
Awesome, thank you!

File: core/src/main/kotlin/org/jetbrains/jewel/IconMapper.kt (hunk 1 above)
Comment (devkanro) [reply core/src/main/kotlin/org/jetbrains/jewel/IconMapper.kt]: Done

**Pull request merged**